.github/CONTRIBUTING.md

```diff
@@ -540,7 +540,113 @@ For more on good git commits, see (among _many_ other):
 
 ### Pull-request
 
-* How is a pull-request that is easily reviewed constructed?
+Why should you care about creating good pull-requests - is it not just a tool for code reviews?
+
+Well first of all, a good pull-request can help reduce how long it takes for your changes to be merged to master by making the life easier for the reviewer, causing a quicker review. 
+
+Secondly a good pull-request is more than just a tool for code reviews - it documents the process of reaching a satisfactory implementation aswell as toughts and discussion surrounding this. 
+
+The rules that applies for writing a [good issue](#issue) also applies for writing a good description for your pull-request.
+But more importantly a pull-request includes commits containing code changes that you want to get merged to master, which means there is a few more points to keep in mind, when constructing a good pull-request. 
+
+Before going through these however - we want to remind you to take ownership of your pull-requests. 
+It is your responsibility as a contributor to get your pull-requests reviewed, therefore reach out and be communicative! 
+
+### The good pull-request solves one issue
```

Comment:
Consider alternatives:

* The good pull-request solves **one single** issue
* The good pull-request solves **a single** issue
* The good pull-request solves **only** one issue

Reply:
Went with the last one :D